<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tb_mostrar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>671</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_db">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Respecto a la base de datos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_db_actualizar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>actualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_db_del">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>borrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_api_leer">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>leer API</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_api">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>respecto al API</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_api_obtener">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>obtener el historia de reportes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_api_verif">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>verificar datos registrados</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_db_del">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
